projects/observability/src/pages/api-trace-detail/api-trace-detail.service.ts

```diff
@@ -66,6 +68,36 @@ export class ApiTraceDetailService implements OnDestroy {
     );
   }
 
+  public fetchLogEvents(): Observable<LogEvent[]> {
+    return this.routeIds$.pipe(
+      switchMap(routeIds =>
+        this.getLogEventsGqlResponse(routeIds.traceId, routeIds.startTime).pipe(
+          map((trace: Trace) => {
+            let logEvents: LogEvent[] = [];
```

Comment:
Can this be shared with the previous service?

Reply:
Done